### PR TITLE
Fix/misc collectible fixes

### LINKED ISSUE #12309: Disable Infura as Collectible provider

### LINKED ISSUE #12338: Alchemy exponential backoff
https://pkg.go.dev/github.com/cenkalti/backoff/v4#section-readme

https://docs.alchemy.com/reference/throughput

### LINKED ISSUE #12352: Disable OpenseaV1 as collectibles provider
https://docs.opensea.io/changelog/api-v2-fully-supported

### PR DESCRIPTION
**Pull request by dlipicar**
Several smallish collectibles changes implemented in separate commits:

- Ensured all chunks are loaded from the same provider
- Exponential backoff for Alchemy requests (Closes https://github.com/status-im/status-desktop/issues/12338)
- Removed Infura client (Closes https://github.com/status-im/status-desktop/issues/12309)
- Removed OpenseaV1 client (Closes https://github.com/status-im/status-desktop/issues/12352)
- Implemented changes to OpenseaV2 API

### CODE REVIEW

**Review by Cuteivist — APPROVED** [review]
LGTM.

Would love second pair of eyes before merge though ;)

**Review by stefandunca — APPROVED** [review]
🤩

**Pull request merged**
